Makefile

```diff
@@ -114,35 +108,36 @@ pytest: compose-build
 	@docker-compose run $(IMAGE_NAME) \
 		pytest $(pytestpath) -m "not integration and not integration_erasure and not integration_external"
 
-# Run the pytest integration tests.
-pytest-integration-access: compose-build
-	@echo "Building additional Docker images for integration tests..."
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml build
-	@echo "Bringing up the integration environment..."
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml up -d
-	@echo "Waiting 20s for integration containers to be ready..."
-	@sleep 20
-	@echo "Running additional setup for mssql integration tests"
-	@docker exec fidesops python tests/integration_tests/mssql_setup.py
-	@echo "Running pytest integration tests..."
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml \
-		run $(IMAGE_NAME) \
-		pytest $(pytestpath) -m integration
```

Comment:
We can add this back for sure. I wasn't sure if it was being used in anyone's workflows, generally because spinning up all the infra to run one test is inefficient. Generally to run one test I'll use `make integration-shell` then run the `pytest ...` commands manually from inside the shell, that way you're not waiting for the infra every time.

Reply:
Update: I've added this back in

Reply:
thank you so much!